litequeue.py

```diff
@@ -185,6 +189,27 @@ def __init__(
         validate_table_name(queue_name)
         self.table = f"[{queue_name}]"
 
+        if run_db_pragmas:
+            # pragmas like journal_mode really only should be run
+            # when the database is first created, and maybe not every time.
+            # They should also be run before tables are created.
```

Comment:
I don't think this is correct, the journal mode can be set/changed at any time. In fact, all the journal models except WAL are specific to the DB connection, not persistent. You can also change the journaling mode after creating the tables.